pkg/util/testappender/internal/dtobuilder/sort.go

```diff
@@ -0,0 +1,73 @@
+package dtobuilder
+
+import (
+	"sort"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+// sortMetricFamilies sorts metric families, the metrics inside them, and the
+// summaries and histograms inside the individual metrics.
+func sortMetricFamilies(mf []*dto.MetricFamily) {
+	sort.Slice(mf, func(i, j int) bool {
+		return mf[i].GetName() < mf[j].GetName()
+	})
+
+	// Sort the met
```

Comment:
nit: `Sort the metrics`, not sure if the comment is needed since the func is called sortMetrics.